en/01_Optuna_Quick_Start.ipynb

```diff
@@ -0,0 +1,358 @@
+    "\n",
+    "# output: evaluation score\n",
+    "print('Mean squared error: ' + str(error))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Performance of Lasso regression is sensitive to the L1 constant, `alpha`, and it's tiresome for humans to manually search this hyperparameter. With Optuna, you can search the appropriate value of `alpha` as follows. Note that you just need to wrap the machine learning logic in the previous cell with `objective` and to set up `study` object."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "zO0XcBNOzuTR"
+   },
+   "outputs": [],
+   "source": [
+    "import optuna  # Don't forget installing optuna with `!pip install optuna`.\n",
+    "import sklearn.datasets\n",
+    "import sklearn.linear_model\n",
+    "import sklearn.metrics\n",
+    "\n",
+    "def objective(trial):\n",
+    "    # hyperparameter setting\n",
+    "    alpha = trial.suggest_uniform('alpha', 0.0, 2.0)\n",
+    "    \n",
+    "    # data loading and train-test split\n",
+    "    X, y = sklearn.datasets.load_boston(return_X_y=True)\n",
+    "    X_train, X_val, y_train, y_val = sklearn.model_selection.train_test_split(X, y, random_state=0)\n",
+    "    \n",
+    "    # model training and evaluation\n",
+    "    model = sklearn.linear_model.Lasso(alpha=alpha)\n",
+    "    model.fit(X_train, y_train)\n",
+    "    y_pred = model.predict(X_val)\n",
+    "    error = sklearn.metrics.mean_squared_error(y_val, y_pred)\n",
+    "\n",
+    "    # output: evaluation score\n",
+    "    return error\n",
+    "\n",
+    "study = optuna.create_study(direction='minimize')\n",
+    "study.optimize(objective, n_trials=20)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "3_C5V0vSHWTV"
+   },
+   "source": [
+    "Let's see the best result among 20 trials."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "42EbIIR5HVD7",
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "print('Minimum mean squared error: ' + str(study.best_value))\n",
+    "print('Best parameter: ' + str(study.best_params))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To access the results of all trials, you can use `study.trials_dataframe`, which shows the details of trials as a pandas dataframe."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "study.trials_dataframe()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "_uruvcV6IraQ"
+   },
+   "source": [
+    "## Imperative Interface: Search Conditional Hyperparameters\n",
```

Comment:
I don't know the reason but this cell is rendered as a code block on colab (see the following snapshot):
![image](https://user-images.githubusercontent.com/181413/60776832-d9400400-a169-11e9-81c5-b111e687743f.png)

It is fine on GitHub:
![image](https://user-images.githubusercontent.com/181413/60776821-cb8a7e80-a169-11e9-8202-4f3cf2c72858.png)

Reply:
In my environment, this problem seems resolved by applying the following diff:
```diff
diff --git a/ja/01_Optuna_Quick_Start.ipynb b/ja/01_Optuna_Quick_Start.ipynb
index 420cb37..92efeb7 100644
--- a/ja/01_Optuna_Quick_Start.ipynb
+++ b/ja/01_Optuna_Quick_Start.ipynb
@@ -414,8 +414,7 @@
   {
    "cell_type": "markdown",
    "metadata": {
-    "colab": {},
-    "colab_type": "code",
+    "colab_type": "text",
     "id": "_uruvcV6IraQ"
    },
    "source": [

```